css/amp.css

```diff
@@ -167,6 +167,24 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
   display: inline-block;
 }
 
+.i-amphtml-blur-loading .i-amphtml-fill-content {
```

Comment:
I think this should be direct descendant `>` to avoid impacting nested children etc.